### PR TITLE
CRI-O jobs: Make CRI-O systemd unit fail if install fails

### PR DESCRIPTION
**Pull request by saschagrunert**
The `;` would make the bash script continue even if the CRI-O install fails. We now fix that by using `&&`.

Beside that we set an absolute path for the `ln` binary.

PTAL @harche @haircommander

### CODE REVIEW

File: jobs/e2e_node/crio/crio-with-1G-hugepages.ign [hunk 1]
```diff
@@ -22,7 +22,7 @@
...
+        "contents": "[Unit]\nDescription=Download and install crio binaries and configurations.\nAfter=network-online.target\n\n[Service]\nType=oneshot\nExecStartPre=/usr/bin/bash -c '\\\n  /usr/bin/curl --fail --retry 5 \\\n    --retry-delay 3 --silent --show-error \\\n    -o /usr/local/crio-nodee2e-installer.sh  \\\n    https://raw.githubusercontent.com/cri-o/cri-o/fa0d058141343ee38b8339ea41f580c361271372/scripts/node_e2e_installer \u0026\u0026 \\\n  /usr/bin/ln -s /usr/bin/runc /usr/local/bin/runc'\nExecStart=/usr/bin/bash /usr/local/crio-nodee2e-installer.sh\n\n[Install]\nWantedBy=multi-user.target\n",
```
Comment (bart0sh) [inline jobs/e2e_node/crio/crio-with-1G-hugepages.ign:25]: Is it intended to use fullwidth ampersands in .ign files? If so, can you explain why?

Comment (saschagrunert) [reply jobs/e2e_node/crio/crio-with-1G-hugepages.ign:25]: Do you mean why it is escaped? Butane generates it that way.

**Pull request merged**